.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "tests/googletest"]
 	path = tests/googletest
 	url = https://github.com/google/googletest.git
+[submodule "ThirdParty/onnx"]
```

Comment:
Should we use upper case for the directory name? I think that all other directories are lower case.

Reply:
@nadavrot It's pretty inconsistent in the code base, e.g., /lib has Backends, ExecutionEngine, etc.
I'm fine either way, thirdparty also would work for me. Let me know.

Reply:
Yes, this sounds good!